DataLayer/BatteryData/BatteryData.cpp

```diff
@@ -27,173 +27,293 @@
 #include <QDebug>
```

Comment:
I don't see where this was being used before, or where it is being used now. Can I remove this?